opennms-webapp/src/test/java/org/opennms/web/svclayer/support/DefaultDistributedStatusServiceTest.java

```diff
@@ -79,6 +80,10 @@
 import junit.framework.AssertionFailedError;
 import junit.framework.TestCase;
 
+/**
+ * Ignore for now, since the remote poller with minion stuff is still in development
+ */
+@Ignore
```

Comment:
Please add an `TODO cpape` so we can find it later on

Reply:
done